### ISSUE TITLE
Element 'field', attribute 'name': The attribute 'name' is not allowed.

### ISSUE THREAD

**Issue by flamboy** [wirecard/magento2-ee#385]
### Prerequisites

- [x] I have my [server set up correctly](https://github.com/wirecard/magento2-ee/wiki/Installation#requirements).
- [x] I am using a [supported version of Magento](https://github.com/wirecard/magento2-ee/wiki#overview-magento-2-shop-plugin).

### Versions

**Magento version:** 2.3.3-p1
**PHP version:** 7.2
**Wirecard module** 3.1.0

### Issue

**Problem description:**

Exception in Magento developer mode
```
1 exception(s):
Exception #0 (Magento\Framework\Exception\LocalizedException): The XML in file "/app/source/vendor/wirecard/magento2-ee/etc/adminhtml/system.xml" is invalid:
Element 'field', attribute 'name': The attribute 'name' is not allowed.
Line: 82
```

**Steps to reproduce:**

1. Set 'developer' mode & clean-up static / caches 
```bin/magento deploy:mode:set developer``` 
2. Try to login in admin panel you will see an exception within XML validation

**Comment by tomazpu** [comment]
Hi @flamboy,

thank you for your report. I will create an issue and we will fix it in the next release.

Best regards, 
Tomaz

**Comment by flamboy** [comment]
It's not so big issue, but every time I switched to developer mode I got this issue. So, I created a temporary patch for this. @tomazpu Thank you!

**Comment by tomazpu** [comment]
@flamboy the fix is ready and will be released in the version 3.1.2 

Best regards,
Tomaz

**Comment by tomazpu** [comment]
Hello @flamboy,

we released the version 3.1.2 in which this issue is fixed. Thank you for the report.

Best Regards,
Tomaz
